_includes/configuration/remove-script-host.md

```diff
@@ -1,6 +1,8 @@
 ## remove_script_host
 
-If this option is enabled the protocol and host part of the URLs returned from the MCFileManager will be removed. This option is only used if the [relative_urls](#relative_urls) option is set to false.
+This option is used if the [`relative_urls`](#relative_urls) option is set to `false` and only applies to links with the same domain as the [`document_base_url`](#document_base_url).
+
+If this option is set to `false`, the `document_base_url` retained for relative links.
```

Comment:
This is worded weirdly and seems to be the opposite of the old explanation? I assume it's false by default, so shouldn't it say something like "If this option is set to true, the document_base_url is removed from relative links"?

Reply:
default is true, so if not defined, the editor will remove the domain etc. the initial description was wrong based on my testing, can you verify from the source?

Reply:
Clarified and fixed, thanks @metricjs